crates/milli/src/update/new/document.rs

```diff
@@ -26,6 +28,7 @@ use crate::{
 ///
 /// The 'doc lifetime is meant to live sufficiently for the document to be handled by the extractors.
 pub trait Document<'doc> {
+    fn geojson_field(&self) -> Result<Option<&'doc RawValue>>;
     /// Iterate over all **top-level** fields of the document, returning their name and raw JSON value.
     ///
     /// - The returned values *may* contain nested fields.
```

Comment:
is _geoJSON also ignored by this function or not?

Reply:
We decided that they should be